### ISSUE TITLE
Printer no longer shows a preview of the part being printed prior to starting a print.

### ISSUE THREAD

**Issue by BrienKing** [prusa3d/Prusa-Firmware-Buddy#3510]
Please, before you create a new bug report, please make sure you searched in open and closed issues and couldn't find anything that matches.

**Printer type** - XL 2TH

**Printer firmware version** -5.1.0 Beta

**Original or Custom firmware** - Original

**Optional upgrades** - None

**USB drive or USB/Octoprint**
USB

**Describe the bug**
Prior to the new (and nice) feature that allows you to select filaments, joining, etc.. it would show you a preview of what you are about to print.   It no longer shows that.

This is an Image I found on the internet that shows the preview I'm referring to.
![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/4928793/0705c148-321c-4232-b21b-778764adc70f)

Now starts on this screen:
![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/4928793/f36bbfed-576b-460b-8af1-ae0fa9e9c495)

**How to reproduce**
Print something....

**Expected behavior**
I want it to show the Part that is about to print and allow me to continue or cancel.  If I continue then it should go to the new filament selection menu.

**Comment by tkircher** [comment]
You have to upgrade PrusaSlicer to generate the correct image file format, now that PNG isn't decoded. It's in the release notes.

**Comment by BrienKing** [comment]
I'm on the latest Slicer and almost latest firmware (don't want to upgrade to the latest firmware because it disables Crash Detection).

**Comment by tkircher** [comment]
> I'm on the latest Slicer and almost latest firmware (don't want to upgrade to the latest firmware because it disables Crash Detection).

Go to Printer Settings > General > Firmware > G-code thumbnails and let me know what formats are listed there

**Comment by BrienKing** [comment]
Well, I never changed it, so not sure why it didn't get updated...

Incorrect settings:
![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/4928793/e7120997-5e7d-4e77-9c05-fb923baf3b50)

Reset to "Default"
![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/4928793/c41984fa-3a64-4fa5-adfb-2c6a12a78e63)

Will have to test with that.

**Comment by carlocarlo91** [comment]
Hi guys, same thing for me with Prusa mini with firmware 5.1.0 and Prusa slicer updated to 2.7.0

![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/151918992/ca1b975b-70a9-4cdc-8c54-a5974bb74436)

I don't see the new format settings

**Comment by VinceKezel** [comment]
I just went to firmware 5.1.0 for my Mini+ and PrusaSlicer 2.7.0 ... and I was confused too, but I see NEWLY sliced and exported files show the thumbnails.

I still think it's a bit of an issue that the display on the Mini+ does not show thumbnails for previous versions of PrusaSlicer exports. I wish it did.

**Comment by sjbrouillard** [comment]
I would agree that it's an issue that the old PNG format is no longer supported. Is there a technical reason that both formats are not supported?

I upgraded to firmware 5.1 for Octoprint support, but am still having some issues there. I kept the output to PNG so that Octoprint file previews would remain in place, but now that I still have to print via USB, the previews have disappeared if I don't re-slice using the new format.

Is there a workaround? Can I output both types of thumbnails?

**Comment by tkircher** [comment]
> I would agree that it's an issue that the old PNG format is no longer supported. Is there a technical reason that both formats are not supported?

There's an explanation here:

https://github.com/prusa3d/Prusa-Firmware-Buddy/releases/tag/v5.1.0

**Comment by sjbrouillard** [comment]
@tkircher - Ugghhh! Clearly my reading comprehension has taken an extreme hit. As they say, everything is a tradeoff. Thanks for pointing out the obvious.

**Comment by epetit91** [comment]
Hi,
with the settings below, the thumbnail is not displayed on the printer screen.
What setting is missing for the thumbnail to display?
Thank you

Printer type - MK4
Printer firmware version -5.1.0
Original firmware 
USB drive
PrusaSlicer 2.7.0
MacOS 10.12.6
![Pict 1](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/7584450/db845615-1859-43f6-b4d9-46c6c8268659)
![Pict 2](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/7584450/3240fadc-c692-4a56-bf93-98d87067fdb2)

**Comment by Yopik23** [comment]
Hi, I have solved missing preview on the printer with saving default profile (Original Prusa MINI & MINI+ Input Shaper (Alpha) as a copy with these setting:
![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/152165282/bfe865f1-dd7e-4243-9577-ff626cf22a58)
 the default one was:
![image](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/152165282/7deb901a-b321-463a-91a4-8acacf1fe3b2)

So you have to rewrite G-Code thumbnails to:
16x16/QOI, 220x124/QOI, 200x240/QOI, 640x480/PNG

Printer type - MINI/MINI+
Printer firmware version -5.1.0
Original firmware 
USB drive
PrusaSlicer 2.7.0

**Comment by BrienKing** [comment]
This is a configuration issue with the printer profile and is not a bug.  Closing.
